plugins/evernote.koplugin/clip.lua

```diff
@@ -235,6 +235,7 @@ function MyClipping:parseHighlight(highlights, bookmarks, book)
             clipping.sort = "highlight"
             clipping.time = self:getTime(item.datetime or "")
             clipping.text = self:getText(item.text)
+            clipping.chapter = item.chapter
             for _, bookmark in pairs(bookmarks) do
                 if bookmark.datetime == item.datetime and bookmark.text then
```

Comment:
(Making this comment here, because it makes it more obvious.)
You set `chapter` on the items in the `highlight` setting - but I get the feeling it makes sense to have it also (or instead, dunno) in `bookmarks`.
Although this could be dealt with by #5794.

Reply:
I set it in highlights, because Evernote plugin operates mostly based on highlights(it uses bookmarks only to get user notes). That storage system is pretty confusing, it's basically duplicating entries for highlights and bookmarks, and duplicating "text" and "notes" in bookmarks. I didn't really see the point of adding this property to bookmarks. It's not a big deal though, I can add it.

Reply:
Actually now that I double checked, it saves it for bookmarks too in `ReaderHighlight:getHighlightBookmarkItem` which is called by `ReaderHighlight:saveHighlight()`.

Reply:
> That storage system is pretty confusing, it's basically duplicating entries for highlights and bookmarks

Yes, it's a mess. #5794 (currently stalled) is an aim at making that saner.